cli/.gitignore

```diff
@@ -31,3 +31,12 @@ node_modules/
 # build folder
 dist/
 bin/
+
+# Jest
+coverage/
+
+# built sc
+*.wasm
+
```

Comment:
Can we add .DS_Store ? It's an annoying mac file

Reply:
No need as it's already in the root gitignore